src/modules/tests/journal-data/candidate/candidate.cat-be.reducer.ts

```diff
@@ -0,0 +1,35 @@
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { createFeatureSelector } from '@ngrx/store';
+import * as candidateActions from './candidate.actions';
+
+export const initialState: CatBEUniqueTypes.Candidate = {
+  candidateId: null,
+  candidateName: {},
+  driverNumber: '',
+  dateOfBirth: '',
+  gender: null,
+  candidateAddress: {},
+  primaryTelephone: '',
+  secondaryTelephone: '',
+  mobileTelephone: '',
+  emailAddress: '',
```

Comment:
could these be nulled if we don't have like the journal data reducer (or does that break stuff)